tests/testthat/test-utils_visR.R

```diff
@@ -4,11 +4,11 @@
 
 # Specifications ---------------------------------------------------------------
 
-#' T1. `align_plots()` accepts a list of `ggplot` objects.
+#' T1. `.align_plots()` accepts a list of `ggplot` objects.
 #' T1.1 No error when a list of `ggplot` objects is passed.
 #' T1.2 An error when nothing or `NULL` is passed.
 #' T1.3 An error when a list containing non-`ggplot` objects is passed.
-#' T2. `align_plots()` aligns multiple `ggplot` objects, taking the legend into account.
+#' T2. `.align_plots()` aligns multiple `ggplot` objects, taking the legend into account.
 #' T2.1 No error when a list of ggplots is passed.
 #' T3. The function `legendopts()` translates the input to a `ggplot2`-compatible list.
```

Comment:
We need a test that checks if the label is not aligned across the plots, but keeps with the individual plots. Perhaps we can make an argument out of it on the longer term similar to cowplot